lint/dummyLen_checker.go

```diff
@@ -0,0 +1,64 @@
+package lint
+
+//! Detects `regexp.Compile*` that can be replaced with `regexp.MustCompile*`.
+//
+// @Before:
+// re, _ := regexp.Compile(`const pattern`)
+//
+// @After:
+// re := regexp.MustCompile(`const pattern`)
+
+import (
+	"go/ast"
+	"go/token"
+
+	"github.com/go-toolsmith/astcopy"
+
+	"github.com/go-toolsmith/astfmt"
+)
+
+func init() {
+	addChecker(&dummyLenChecker{}, attrSyntaxOnly, attrExperimental)
```

Comment:
I don't like `dummy` name part.
It's too generic and doesn't seem to describe what checker does too well.
Quite hard to give right name from the first try, but maybe proper summary section can help us to find it.